conanfile.txt

```diff
@@ -1,4 +1,5 @@
 [requires]
+vehicle-model/generated
```

Comment:
is this the correct version? seems a bit odd to me

Reply:
Since its an auto-generated package `generated` is indeed the version specifier of the vehicle model